src/models/WidgetEventCapability.ts

```diff
@@ -56,6 +56,37 @@ export class WidgetEventCapability {
         return false;
     }
 
+    public static forStateEvent(
+        direction: EventDirection,
+        eventType: string,
+        stateKey?: string,
+    ): WidgetEventCapability {
+        // TODO: Enable support for m.* namespace once the MSC lands.
```

Comment:
Is there an issue tracking this?

Reply:
There will be :D

Reply:
// https://github.com/matrix-org/matrix-widget-api/issues/22